packages/datadog-instrumentations/src/child_process.js

```diff
@@ -63,14 +86,37 @@ function wrapChildProcessSyncMethod (shell = false) {
 
       const innerResource = new AsyncResource('bound-anonymous-fn')
       return innerResource.runInAsyncScope(() => {
-        return childProcessChannel.traceSync(
-          childProcessMethod,
-          {
-            command: childProcessInfo.command,
-            shell: childProcessInfo.shell
-          },
-          this,
-          ...arguments)
+        const context = {
```

Comment:
couldn't you use `childProcessInfo` as `context`?

Reply:
Reasons why I decided not using `childProcessInfo` as context:
- In the previous version, it wasn't, it was a subset of properties of `childProcessInfo`, so I maintained it.
- I can't use directly the same object instance, (you can see that I am creating new `{ ...context, abortController }` each time, so less properties the best.